main.go

```diff
@@ -173,7 +173,7 @@ func getVariableKey(source string) (key, value string) {
 	if strings.Contains(k, ":") {
 		re := strings.Split(k, ":")
```

Comment:
strings.SplitN(k, ":", 2) 
It's better？

Reply:
yes